HopsanGUI/Utilities/GUIUtilities.cpp

```diff
@@ -381,6 +381,12 @@ QVector< complex<double> > realToComplex(const QVector<double> &rRealVector)
 //! @param [out] Cb Frequency compensation factor
 void windowFunction(QVector<double> &data, WindowingFunctionEnumT function, double &rCa, double &rCb)
 {
+    if(data.empty()) {
```

Comment:
Inst the problem here if data has size  1

Reply:
You are right. I changed the check so that data size must be greater than 1.